pkg/api/datasources.go

```diff
@@ -513,3 +514,94 @@ func (hs *HTTPServer) CheckDatasourceHealth(c *models.ReqContext) response.Respo
 
 	return response.JSON(200, payload)
 }
+
+// Get /api/datasources/uid/:uid/history
+func (hs *HTTPServer) GetDataSourceHistory(c *models.ReqContext) response.Response {
```

Comment:
Should there be a check for `!hs.VCS.IsDisabled()` ?

Reply:
I just realized you don't have it until we merge my PR

Reply:
Good idea but I need to wait for your PR to be merged to make this change